src/main/java/TaskList.java

```diff
@@ -0,0 +1,188 @@
+import java.io.*;
```

Comment:
Perhaps the relevant `io` files could be listed and imported explicitly instead of using a wildcard import?

Reply:
Will do!